packages/lesswrong/server/notificationTypesServer.tsx

```diff
@@ -97,6 +99,22 @@ export const NewShortformNotification = serverRegisterNotificationType({
   }
 })
 
+export const NewTagPostsNotification = serverRegisterNotificationType({
+  name: "newTagPosts",
+  canCombineEmails: true,
+  emailSubject: ({user, notifications}) => {
+    const [documentId, documentType] = notifications[0]
+    return taggedPostMessage({documentId, documentType})
+  },
+  emailBody: ({user, notifications}) => {
+    const [documentId, documentType] = notifications[0]
```

Comment:
Since this has `canCombineEmails: true`, `notifications` may be an array of multiple notifications. So the email needs to contain all of them.

Reply:
Ah, I think probably this should just have combineEmails: false, although not sure.